test/unit/raptor/RaptorDepartAfterQuery.spec.ts

```diff
@@ -29,6 +29,98 @@ describe("RaptorDepartAfterQuery", () => {
     ]);
   });
 
+  it("finds route not through via simple", () => {
+    const trips = [
+      t(st("A", null, 1000),
+        st("B", 1025, 1028),
+        st("C", 1100, null)),
+      t(st("A", null, 1010),
+        st("D", 1030, 1035),
+        st("C", 1110, null))
+    ];
+
+    const raptor = RaptorQueryFactory.createDepartAfterQuery(trips, {}, {}, calendars, journeyFactory);
+    const result = raptor.plan("A", "C", new Date("2018-10-16"), 900, ["B"]);
+
+    setDefaultTrip(result);
+
+    chai.expect(result).to.deep.equal([
+      j([
+        st("A", null, 1010),
+        st("D", 1030, 1035),
+        st("C", 1110, null)
+      ])
+    ]);
+  });
+
+  it("finds route not through via", () => {
+    const trips = [
+      t(st("A", null, 1000),
+        st("B", 1025, 1028),
+        st("C", 1100, 1108),
+        st("D", 1135, null)),
+      t(st("A", null, 1000),
+        st("E", 1035, 1040),
+        st("C", 1050, null)
+        )
+    ];
+
+    const raptor = RaptorQueryFactory.createDepartAfterQuery(trips, {}, {}, calendars, journeyFactory);
+    const result = raptor.plan("A", "D", new Date("2018-10-16"), 900, ["B"]);
+
+    setDefaultTrip(result);
+
+    chai.expect(result).to.deep.equal([
+      j([
+        st("A", null, 1000),
+        st("E", 1035, 1040),
+        st("C", 1050, null)
+      ],
+      [
+        st("C", 1100, 1108),
+        st("D", 1135, null)
+      ])
+    ]);
+  });
+
+  it("finds correct not via ignoring transfers", () => {
+      const trips = [
+          t(st("A", null, 1000),
+              st("B", 1025, 1028),
```

Comment:
Is there a way of easily representing a train that goes through a station, but doesn't stop at that station?

Reply:
You can create a StopTime that has a false pickUp and dropOff.

What test did you have in mind? I think you still should not be allowed to take the trip that would go through the station regardless if you can or cant get on or off of it.

Reply:
I agree that a not-via should prevent you from going through that station, even with no stop.

I think create a new function in the utils class like `via`, similar to `st`, have it generate stops with optional pick ups or drop offs. Then add a test making sure that it finds the correct journeys.

Reply:
7fb6b84 added new via util function and test.

Reply:
Cool, looks good!